.travis/now

```diff
@@ -0,0 +1,141 @@
+#
+# Rose is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Rose is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Rose. If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+
+# Bash script for use with Travis-CI builds.
+
+set -o xtrace
+
+if [[ -f "${HOME}/.bashrc" ]]; then
+    cat "${HOME}/.bashrc"
+    source "${HOME}/.bashrc"
+fi
+
+set -eu
+
+APT=()
+PIP=()
+NPM=()
+RC_PATH=("${HOME}")
+
+join () {
+    local IFS="$1";
+    shift;
+    echo "$*";
+}
+
+install_coverage () {
+    export COVERAGE_PROCESS_START="${TRAVIS_BUILD_DIR}/.coveragerc"
+    export PYTHONPATH="${TRAVIS_BUILD_DIR}/.travis:${PYTHONPATH:-}"
+    PIP+=(coverage pytest-cov)
+}
+
+install_rose () {
+    export PYTHONPATH="lib/python/:${PYTHONPATH:-}"
+    RC_PATH+=("${PWD}/bin")
+}
+
+install_cylc () {
+    wget 'https://github.com/cylc/cylc/archive/master.tar.gz' \
+        -O '/tmp/cylc-master.tar.gz'
+    tar -xvf '/tmp/cylc-master.tar.gz' -C "${HOME}"
+    APT+=(at heirloom-mailx)
+    PIP+=(colorama python-jose pyzmq Jinja2)
+    RC_PATH+=("${HOME}/cylc-master/bin")
+}
+
+install_fcm () {
+    sudo sh -c 'echo "deb http://opensource.wandisco.com/ubuntu \
+        `lsb_release -cs` svn19" >> /etc/apt/sources.list.d/subversion19.list'
+    sudo wget -q http://opensource.wandisco.com/wandisco-debian.gpg -O- | \
+        sudo apt-key add -
+    wget 'https://github.com/metomi/fcm/archive/master.tar.gz' \
+        -O '/tmp/fcm-master.tar.gz'
+    tar -xvf '/tmp/fcm-master.tar.gz' -C "${HOME}"
+    APT+=(subversion build-essential gfortran libxml-parser-perl \
+          libconfig-inifiles-perl libdbi-perl libdbd-sqlite3-perl)
+    RC_PATH+=("${HOME}/fcm-master/bin")
+}
+
+install_sphinx () {
+    APT+=(latexmk texlive texlive-generic-extra texlive-latex-extra \
+          texlive-fonts-recommended graphviz)
+    PIP+=(sphinx sphinx_rtd_theme sphinxcontrib-httpdomain hieroglyph)
+}
+
+install_unittests () {
+    PIP+=(pytest)
+}
+
+install_linters () {
+    PIP+=(pycodestyle)
+    NPM+=(eslint)
+}
+
+install () {
+    for arg in "$@"; do
+        "install_${arg}"
+    done
+
+    if [[ ${#APT[@]} > 0 ]]; then
+        sudo apt-get update
+        sudo apt-get install -y "${APT[@]}"
+    fi
+
+    if [[ ${#PIP[@]} > 0 ]]; then
+        pip install "${PIP[@]}"
+    fi
+
+    if [[ ${#NPM[@]} > 0 ]]; then
+        npm install -g "${NPM[@]}"
+    fi
+
+    cat >"${HOME}/.bashrc" <<<"export PATH='$(join ':' "${RC_PATH[@]}"):$PATH'"
+}
```

Comment:
So, after a bit of bodge testing this change is what was causing the tests to fail which is odd since looking at the `.bashrc` is seems pretty innocuous.

Looks like this is the route we went down before as well: https://github.com/metomi/rose/blob/2019.01.x/.travis.yml#L16

I think its a `PS1` trap at the start which is causing the issue. Perhaps we could prepend to the bashrc.